amacc.c

```diff
@@ -2101,4 +2084,4 @@ int main(int argc, char **argv)
     return ret;
 }
 
-// vim: set tabstop=4 shiftwidth=4 expandtab:
+// vim: set tabstop=4 shiftwidth=4 expandtab:
```

Comment:
Don't change this line.

Reply:
Uh? This is the last comment inside amacc.c. The source line is reduce.